worker/modelworkermanager/modelworkermanager.go

```diff
@@ -171,10 +180,6 @@ func (m *modelWorkerManager) Wait() error {
 }
 
 func (m *modelWorkerManager) loop() error {
-	controllerConfig, err := m.config.Controller.Config()
-	if err != nil {
-		return errors.Annotate(err, "unable to get controller config")
-	}
```

Comment:
Notice how this was only retrieved at the beginning of the loop and not every time the model starts. This controller config was used for log rotation https://github.com/juju/juju/blob/9589eb7adc26ea8010dd968284476ca72ff4a009/cmd/jujud/agent/machine.go#L1039-L1040